### DISCUSSION TITLE
Industries & Sectors

### DISCUSSION THREAD

**Discussion by ccrvlh** [JerBouma/FinanceDatabase#21] [Q&A] [answered]
@JerBouma 
We have been looking for structured fundamental data for a while, and it's been a hustle. One of our biggest challenges is finding financials with structured (and standardized) industries & sectors (in our internal DB we use both GICS and ISIC).

Wondering what kind of sources have you analyzed before that might/might not contain this information in a structure manner.
It can be an option to "map" GICS / ISICS to Yahoo's classification, this wouldn't be our first option, since it would generate higher maintenance work to update the mapping, even though it shouldn't change much.

If at the end of the day, Yahoo's ends up being the easiest, we could contribute to the mapping, since we will keep our GICS/ISIC database anyways.

Any knowledge you could share would be great.
Thanks a lot for sharing your work on this!

Best

**Comment by JerBouma** [comment]
Good day!

Since Yahoo Finance is such an easy source to acquire data from and their data is very extensive, including many exchanges, this has always been my go-to. I am aware that the mapping is unfortunately not GICS and I did think about mapping it myself but that's a bit of a hassle and too much effort for now.

If you could contribute to the mapping that would be beneficial. In the end it just requires a file that translates certain industries or sectors to the correct GICS category. It can just be a bit tedious to do right.

**Reply by ccrvlh** [reply]
Fair enough. For some reason I've always been a bit suspicious of YF's data, but to be fair this is totally unfounded from my part.
We'll definitely take a look at this. The good part is that Yahoo's classification rather shallow (sector > industry) while both GICS/ISIC have 4-6 levels if I'm not mistaken, so there's not that much data to map. Keep you posted if we end up going down this path.
Thanks!

**Reply by JerBouma** [reply]
So, the database doesn't really extract any fundamentals and whatnot. So, even though the data can not be as of high quality as you like, it doesn't matter too much because it is mostly about the ticker and the categorisation of the ticker. Then, you should be able to use the categorisation with preferred data providers.

Thank you, happy to help out if needed!

**Comment by JerBouma** [comment, marked as answer]
I've now successfully transformed everything to a loose approximation of GICS! 🎊
